packages/slate/src/transforms/node.ts

```diff
@@ -1,3 +1,4 @@
+import { update } from 'lodash'
```

Comment:
Ah it looks like we aren't using `update` yet in this code, maybe it's in code you haven't pushed yet?

Reply:
I didn't use it too 🤔